### ISSUE TITLE
Hue Bridge Configuration Function Not Working

### ISSUE THREAD

**Issue by jjjjjjjj99** [isaacrlevin/presencelight#784]
When trying to configure connection to my Hue Bridge behavior is not consistent/as expected.

If I ask the app to 'find my hue bridge' I receive no feedback/response.

If I enter the IP manually and click "find hue bridge" it does not provide any response.

After entering IP manually If I click to another configuration tab like LIFX and then click back to Hue tab it now shows "Connect to Hue" and allows me to choose from Hue "Lights".  App will work in this manner.

In my case I have 2 hue strips configured as a hue group that I like to have the app control.  When I click the button for "Get Hue Groups" I receive the following: "Error Occurred Connect to Hue, please try again."

Running 5.5.17 on Windows 11 Enterprise Build 22621

![image](https://user-images.githubusercontent.com/18564207/210441946-4327e85a-1d5e-44fc-a3c3-7b924817ec55.png)

[log-2023010316.txt](https://github.com/isaacrlevin/presencelight/files/10339811/log-2023010316.txt)

**Comment by isaacrlevin** [comment]
What kind of bridge do you have? Does it show up in the Hue app on your phone?

**Comment by jjjjjjjj99** [comment]
I have the v2 Hue bridge.  My hue app can see and address each hue light individually or as a group (room).  The PresenceLight app can see each light individually and can update 1 light at a time.  For the last year plus I've had it configured to control the group (room) which controls 2 hue led strips.  When I had to reset my config recently now I can no longer get the 'get hue groups' function to work as mentioned above.

**Comment by isaacrlevin** [comment]
I am currently working on getting a new build. I have some backend/infra issues but will update when it is out there.

**Comment by isaacrlevin** [comment]
@jjjjjjjj99 
Can you try the latest nightly?

**Comment by jjjjjjjj99** [comment]
> @jjjjjjjj99 Can you try the latest nightly?

Sorry for the delay. I loaded the nightly build 5.5.41 today and behavior is better, but not sure it is fully resolved.  Now when I ask it to ("Get Hue Groups" using a local IP connection) I see a list of my groups vs the error message mentioned above.  However the first half a dozen times I tried to select a group it would not let me click on any group in the drop down list.  After I came to type a reply to you suddenly in the background it happened to 'pick' one of my groups and just happened to be the one I want sync to Teams... so now the sync is working for the moment as design, but not entirely sure how it happened.

The 'use remote api' still is not working for your awareness.  It authenticates me to the Philips website, but does not allow me to see/control any of my lights or groups (similar error message to above).
